### DISCUSSION TITLE
Docker部署扫码提示密码不正确或账号被冻结是为什么啊？

### DISCUSSION THREAD

**Discussion by m0eak** [SinKy-Yan/zhenxunbot-docker#29] [Q&A]
![image](https://user-images.githubusercontent.com/50049180/228280286-f32c685b-4cd3-4523-b344-fdff78db0a2d.png)

**Comment by SinKy-Yan** [comment]
不知道，你去问gocq

**Comment by m0eak** [comment]
gocq是啥？--发自新浪邮箱客户端
在 3月28日 23:00，源源球球✨ ***@***.***> 写道：

不知道，你去问gocq

&mdash;
